fe-test-files-check-action/src/findUntestedFiles.js

```diff
@@ -3,6 +3,27 @@ const minimatch = require('minimatch');
 const path = require('path');
 const core = require('@actions/core');
 
+const DONT_NEED_TESTS_JS_FILE_NAME_KEYWORDS = [
```

Comment:
好像可以把 JS 拿掉，因為也有 TS？
```suggestion
const FILE_NAME_KEYWORDS_NOT_REQUIRING_TESTS = [
```